lib/scrum/priority_name.rb

```diff
@@ -0,0 +1,15 @@
+module Scrum
+  class PriorityName
+    PRIORITY_REGEX      = /^(?:\([\d.]+\) )?P(\d+): /
```

Comment:
I guess it shouldn't match dot, but maybe someone is using fractions, like `(1.5) P2:`.
So, an improvement might be `/^(?:\(\d+(?:\.\d+)?\) )?P(\d+): /`?

Anyways, I just extracted these methods and don't plan to improve them with this PR. Additionally `priority` should be a private method, there are no consumers outside `PriorityName`, but since there were already some tests for it I left it public.